test/text_tags_test.js

```diff
@@ -400,4 +400,19 @@ describe('TextTags', function() {
       assert.equal("My Title", seo.getTitle(config, 'home-page', {data: {title: "My Title"}}, {}));
     });
   });
+
+  describe('Tag Page', function() {
+    it("Generates SEO tags for a tag page", function () {
+      const seoConfig = {
+        generators: [TextTags],
+      }
+      const tag = {slug: "Foobar", description: "Some Foobar", name: "Footag"}
+      const string = getSeoMetadata(seoConfig, {"sketches-host": "http://foo.com"}, 'tag-page', {data: {tag: tag}}, {url: url.parse("/tag-page")})
+      assertContains('<title>Footag</title>', string);
+      assertContains('<meta name="description" content="Some Foobar"/>', string);
+      assertContains('<meta name="keywords" content="Footag"/>', string);
+      assertContains('<link rel="canonical" href="http://foo.com/tag-page"/>', string);
```

Comment:
as discussed send the url from the publisher, since publisher is the one setting the topic page url. 
cc: @amithgeorge